C/src/modal_models/FurutaPendulum/FurutaPendulum.lf

```diff
@@ -23,40 +23,24 @@ target C {
     timeout: 3 secs,
     fast: true,
     flags: "-lm",
-    build: "./build_run_plot.sh FurutaPendulum pendulum open"
+    build: "./build_run_plot.sh FurutaPendulum open"
```

Comment:
This doesn't work on Linux platforms and will lead to frustration with folks who don't understand why. We should check for the platform in the script. I could push a fix for that to this branch.

Reply:
That would be great

Reply:
I just added a platfom-specific switch to the script that will use the Linux xdg-open command.
It works for me on Linux but I do not have a Mac at hand, so I don't know if the script still works there. Could you test that?

Reply:
I am unable to compile this using the master branch. I get this:
```
Undefined symbols for architecture x86_64:
  "_lf_critical_section_enter", referenced from:
      __lf_schedule_token in libcore.a(reactor_common.c.o)
      __lf_schedule_copy in libcore.a(reactor_common.c.o)
      __lf_schedule_value in libcore.a(reactor_common.c.o)
      _next in libcore.a(reactor.c.o)
  "_lf_critical_section_exit", referenced from:
      __lf_schedule_token in libcore.a(reactor_common.c.o)
      __lf_schedule_copy in libcore.a(reactor_common.c.o)
      __lf_schedule_value in libcore.a(reactor_common.c.o)
      _next in libcore.a(reactor.c.o)
  "_lf_notify_of_event", referenced from:
      __lf_schedule_token in libcore.a(reactor_common.c.o)
      __lf_schedule_copy in libcore.a(reactor_common.c.o)
      __lf_schedule_value in libcore.a(reactor_common.c.o)
  "_lf_sched_init", referenced from:
      __lf_initialize_trigger_objects in FurutaPendulum.c.o
ld: symbol(s) not found for architecture x86_64
```

Reply:
I initially tested the model with the latest release and it worked but now that I use the latest master I run into the same error. I will take a look at it.

Reply:
Ok, I was able to fix it. Apparently the `cmake` command now requires a bunch of defines in order to work correctly, but I do not know which caused the error in the first place. I simply copied the definitions from the default "build and run" and now it works for me, both on master and the most recent release.
Can you confirm this?

Reply:
Confirming that this works!